packages/expect/src/index.js

```diff
@@ -314,19 +314,34 @@ const _validateResult = result => {
   }
 };
 
+function assertions(expected: number) {
+  const error = new Error();
+  if (Error.captureStackTrace) {
+    Error.captureStackTrace(error, assertions);
+  }
+
+  getState().expectedAssertionsNumber = expected;
+  getState().expectedAssertionsNumberError = error;
+}
+function hasAssertions(...args) {
```

Comment:
has to be a flow bug that I was only yelled at after extracting this (complaining that `hasAssertions` takes args when the type says no args)

Reply:
Curious what caused this (would be great to setup a repro for Flow folks). But `...args` seems ok here.

Reply:
Yeah, unable to reproduce in their repl...